### ISSUE TITLE
The number of image pairs used for training on the ME-v2 dataset

### ISSUE THREAD

**Issue by McFly-Jia** [KevinJ-Huang/ExposureNorm-Compensation#1]
It is mentioned in the paper that ME-v2 dataset selects the middle exposure subset in ME dataset as the ground truth and preserves the other exposure subsets as the multiple exposure input. It is also mentioned in the paper that ME-v2 dataset includes 14,144 pairs for training. How is 14,144 obtained and why isn't the number of image pairs used for training 17675/5*4=14140?

**Comment by KevinJ-Huang** [comment]
Thanks for your reminder! 14,144 is a typo because I have included the four folders as image samples, so you are right. Thanks again for your reminder, I will fix this typo in readme.md.

**Comment by McFly-Jia** [comment]
In Table 1, are the PSNR and SSIM on ME dataset computed by using redered images by Expert C as GT?

**Comment by KevinJ-Huang** [comment]
Yes, you can find the GT in the target folder of this dataset

**Comment by McFly-Jia** [comment]
This dataset has 5  target folders

**Comment by KevinJ-Huang** [comment]
you are right, I choose the redered images by Expert C as GT, I will add this in the readme.md
